.travis.yml

```diff
@@ -14,27 +14,35 @@ branches:
 
 sudo: true
 
-php:
-  - 7.0
-  - 7.1
-  - hhvm
-
 matrix:
   fast_finish: true
   include:
+    - php: 7.0
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
+    - php: 7.0
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
+    - php: 7.0
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
     - php: 7.1
-      env: ES_VERSION="5.0" TEST_BUILD_REF="origin/5.0"
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
     - php: 7.1
-      env: ES_VERSION="5.1" TEST_BUILD_REF="origin/5.1"
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
     - php: 7.1
-      env: ES_VERSION="5.2" TEST_BUILD_REF="origin/5.2"
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
+    - php: hhvm
```

Comment:
https://github.com/elastic/elasticsearch-php/issues/611 needs to be resolved first